app/src/main/java/org/listenbrainz/android/presentation/features/dashboard/DashboardBackdropBackLayerContent.kt

```diff
@@ -122,6 +125,26 @@ fun BackLayerContent(activity: Activity, applicationContext: Context) {
                     }
                 }
             }
+            
+            // Yim Card
+            Card(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(18.dp)
+                    .clip(RoundedCornerShape(16.dp))
+                    .clickable(onClick = {
+                        activity.startActivity(
+                            Intent(
+                                applicationContext,
+                                YearInMusicActivity::class.java
+                            )
+                        )
+                    }),
+                elevation = 0.dp,
+                backgroundColor = MaterialTheme.colors.onSurface
+            ){
+                Text(text = "Yim Activity Test", color = Color.White, modifier = Modifier.padding(16.dp))
```

Comment:
Can we fix this in this PR itself?

Reply:
"Your Year in Music" could be the title and the usual logo could be beside it

Reply:
My bad, forgot about this one. I was just waiting for aerozol to get me some logo to display on the card.

Reply:
Let's do the official YIM logo in the next PR. For now, a placeholder is fine

Reply:
Alright!